packages/cactus-plugin-keychain-azure-kv/src/main/typescript/generated/openapi/typescript-axios/.gitignore

```diff
@@ -0,0 +1,4 @@
+wwwroot/*.js
```

Comment:
@jagpreetsinghsasan Please ignore this file in the openapi generator ignore file (and also remove the file itself from version control)